### PR TITLE
adds Qr code

### LINKED ISSUE #68: Generate QR code images for short URLs
As in Firefly 1.x, add an action for each item so you can download a QR code that contains the short URL.

### PR DESCRIPTION
**Pull request by volgorean**
adds Qr code for short links, image link icon in dash, and corresponding tests.
closes #68

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,12 +9,14 @@ gem 'haml'
...
 
 gem 'sqlite3'
+gem 'rqrcode'
 
 group :development do
   gem 'byebug'
 end
 
 group :test do
+  gem 'rspec'
```
Comment (ariejan) [inline Gemfile]: What's your reason for adding rspec to the project? Minitest is already available.

**Comment by ariejan** [comment]
Looks good! Aside from adding rspec :-) I'll migrate your tests to minitest during lunch and merge this. Thanks!

**Comment by ariejan** [comment]
Handling this in #78.

**Comment by jvanbaarsen** [comment]
@ariejan That was an early lunch :trollface:

**Comment by ariejan** [comment]
@jvanbaarsen call it morning coffee, then :-)

File: Gemfile (hunk 1 above)
Comment (volgorean) [inline Gemfile]: assert page.has_css? "img[src*='/admin/items/#{@items[7].id}.png']"

I was unable to easily test for an image on the page.

**Pull request closed without merging**